source/atlas-cli-deploy-local.txt

```diff
@@ -0,0 +1,894 @@
+                    {
+                      plot: 'A trio of guys try and make up for missed opportunities in childhood by forming a three-player baseball team to compete against standard children baseball squads.',
+                      title: 'The Benchwarmers'
+                    },
+                    {
+                      plot: 'A young boy is bequeathed the ownership of a professional baseball team.',
+                      title: 'Little Big League'
+                    },
+                    {
+                      plot: 'A trained chimpanzee plays third base for a minor-league baseball team.',
+                      title: 'Ed'
+                    },
+                    {
+                      plot: 'The story of the life and career of the famed baseball player, Lou Gehrig.',
+                      title: 'The Pride of the Yankees'
+                    },
+                    {
+                      plot: 'Babe Ruth becomes a baseball legend but is unheroic to those who know him.',
+                      title: 'The Babe'
+                    }
+                  ]
+
+Perform a Vector Search on a Local MongoDB Deployment
+-----------------------------------------------------
+Use the ``atlas deployments`` command to perform a vector search. To 
+learn more, see 
+:atlas:`How to Index Vector Embeddings for Vector Search 
+</atlas-search/field-types/knn-vector/>`.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create an ``indexDef-vector.json`` configuration file.
+
+      .. code-block:: json
+
+         {
+           "collectionName": "embedded_movies",
+           "database": "sample_mflix",  
+           "mappings": {
+             "dynamic": true,
+             "fields": {
+               "plot_embedding": {
+                 "type": "knnVector",
+                 "dimensions": 1536,
+                 "similarity": "euclidean"
+               }
+             }
+           },
+           "name": "vectorSearchIndex"
+         }
+   
+   .. step:: Create a local |fts| index.
+
+      a. Run the following command:
+
+         .. code-block:: sh
+
+            atlas deployments search indexes create --file indexDef-vector.json
+
+      #. Specify the deployment and press ``Enter``.
+
+   .. step:: Connect to the deployment.
+
+      a. Run the following command to connect to a deployment:
+
+         .. code-block:: sh
+
+            atlas deployments connect
+      
+      #. Specify the deployment to connect to and press 
+         ``Enter``.
+
+      #. Specify ``mongosh`` and press ``Enter``.
+
+   .. step:: Switch to the ``sample_mflix`` database.
+
+      .. code-block:: sh
+
+         use sample_mflix
+
+   .. step:: Run an example query.
+
+      .. io-code-block::
+
+         .. input::
+            :language: sh
+            
+            db.embedded_movies.aggregate([
+              {
+                "$search": {
+                  "index": "vectorSearchIndex",
+                  "knnBeta": {
```

Comment:
can we use https://preview-mongodbkanchanamongodb.gatsbyjs.io/cloud-docs/DOCSP-32399/atlas-vector-search/vector-search-stage/#examples